src/main/scala/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.scala

```diff
@@ -87,14 +88,16 @@ private[bigquery] class DirectBigQueryRelation(
           .setRequestedStreams(getNumPartitions)
           .setReadOptions(readOptions)
           .setTableReference(tableReference)
+          // TODO(aryann): Once we rebuild the generated client code, we should change this to
+          // use setShardingStrategy().
```

Comment:
Are you sure this won't get stripped out for less authorized users?

Are you sure this isn't released yet? We could try updating the dependencies. 

Please document a little clearer it's more than a little scary as is.

Are you planning to support not balanced going forward? It might be nice to give users the choice (and still turn it on by default)

Reply:
Summarizing our earlier conversation for anyone following along: this is just a client-side issues because the generated clients haven't been updated. The BALANCED strategy is public and anyone who knows how to specify it on the wire can use it. I think in most cases, you want BALANCED. LIQUID is not particularly useful as most data processing systems want bounded sources.